apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTheme } from "styled-components/native";
+import React, { useState } from "react";
+import { Linking } from "react-native";
+import { BottomDrawer, Button, Flex, Icons, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { urls } from "~/utils/urls";
+
+export function NeedMemoTagModal() {
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = useState(false);
+  const openModal = () => setIsOpen(true);
+  const closeModal = () => setIsOpen(false);
+
+  return (
+    <>
+      <Button type="accent" onPress={openModal}>
+        {t("transfer.receive.memoTag.link")}
+      </Button>
+
+      <BottomDrawer isOpen={isOpen} onClose={closeModal}>
+        <Flex alignItems="center" mb={7}>
+          <Flex
```

Comment:
Not sure but can't we use `Circle` from  "~/components/Circle";
 for this ?

Reply:
Yes that's better. I had only looked in the `native-ui`. Thanks